### PR TITLE
fix issues found in smoke test

### PR DESCRIPTION
**Pull request by ravishetye**
- [x] smoke test
- [x] client tests : pending

### CODE REVIEW

File: webapp/client/src/features/datasetVersions/store/actions.ts [hunk 1]
```diff
@@ -114,7 +114,7 @@ export const loadDatasetVersion = (
   dispatch(action(loadDatasetVersionActionTypes.REQUEST, { id }));
 
   await ServiceFactory.getDatasetVersionsService()
-    .loadDatasetVersion(id, datasetId)
+    .loadDatasetVersion(workspaceName, id, datasetId)
```
Comment (ravishetye) [inline webapp/client/src/features/datasetVersions/store/actions.ts:117]: @CaptEmulation : this fixes a issue i pinged on #webapp on slack , not sure if it need a better treatment, if so feel free to commit to the PR.

**Comment by ravishetye** [comment]
@ad-47 : if you get time please review the PR, we want to publish OSS builds on Monday and needed to add back some dataset related code to master due to the delayed migration.

File: webapp/client/src/features/datasetVersions/store/actions.ts (hunk 1 above)
Comment (CaptEmulation) [reply webapp/client/src/features/datasetVersions/store/actions.ts:117]: THANK YOU. I was looking at the exact line and scratching my head very recently

Comment (CaptEmulation) [reply webapp/client/src/features/datasetVersions/store/actions.ts:117]: well actually I was looking at another usage of this method where it was not including the `datasetId`.

In any case, this LGTM :+1:

**Review by CaptEmulation — APPROVED** [review]
LGTM

**Pull request merged**